setup.cfg

```
@@ -67,11 +67,11 @@ server=
     django-filter
     django-health-check
     # dynaconf 3.1.3 had a regression https://github.com/ansible-community/ara/issues/212
-    # dynaconf dropped support for py35 at version 3.0.0: https://github.com/ansible-community/ara/issues/370
-    dynaconf[yaml]>=3.0.0,!=3.1.3,<4.0.0;python_version>='3.6'
-    dynaconf[yaml]<3.0.0;python_version<'3.6'
+    dynaconf[yaml]>=3.0.0,!=3.1.3,<4.0.0
     tzlocal<3.0
-    whitenoise
+    # whitenoise 6.0.0 dropped support for py36
+    whitenoise<6.0.0;python_version=='3.6'
+    whitenoise;python_version>'3.6'
```

@dmsimard shoudn't this version pinned to python 3.7 ? [c.f](https://github.com/evansd/whitenoise/blob/c82f9a3606074d53063243308c2a9afdbe0195cb/setup.cfg#L40)
```suggestion
    whitenoise;python_version>='3.7'
```

---

@DataHearth I will double check and test to make sure, though my understanding is that ``>3.6`` and ``>=3.7`` are functionally equivalent in this context.

---

sure !

---

I've rebased with your suggestion included, let's go with that. Thanks :)